GOVERNANCE.md

```diff
@@ -0,0 +1,74 @@
+# kyverno Governance
+
+This document defines governance policies for kyverno project.
+
+## Principles
+
+The kyverno project community adheres to the following principles:
+
+- Open: The kyverno community strives to be open, accessible and welcoming to everyone. Anyone may contribute, and contributions are available to all users according to open source values and licenses.
+- Transparent and accessible: Any changes to kyverno source code and collaborations on the project are publicly accessible (GitHub issues, PRs, and discussions).
+- Merit: Ideas and contributions are accepted according to their technical merit and alignment with project objectives, scope, and design principles.
+
+
+## Code of Conduct
+
+kyverno follows [Code of Conduct](./CODE_OF_CONDUCT.md), which is aligned with the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+
+
+## Contributors
```

Comment:
Can we change this to single `Roles` section and point to the existing docs at https://kyverno.io/community/#project-roles? 

> The Kyverno project welcomes all contributor and has well defined roles specified at  https://kyverno.io/community/#project-roles.

In general, we want to avoid/limit duplication, so its easier to maintain the guidelines. Please check other sections as well, for opportunities to de-duplicate.

The other option is to make this document the source, and update the docs to point here.

Reply:
Check Now sir